Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.12-slim
```

Comment:
Ensure that the base image version `python:3.12-slim` is acceptable according to the task requirements. The previous review suggested using `python:3.10.8-slim`. Please confirm if this version is suitable for your project.